source/index.html

```diff
@@ -1,12 +1,149 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Кэт энерджи</title>
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page">
+
+  <header class="main-header">
+    <a class="main-header__logo logo">
+      <img class="logo__image" src="img/logo-image.svg" alt="Логотип Кэт энерджи" width="60" height="50">
+      <img class="logo__text" src="img/logo-text.svg" alt="Кэт энерджи">
+    </a>
+    <nav class="main-nav">
+      <ul class="main-nav__list site-list">
+        <li class="site-list__item">
+          <span class="site-list__navigation site-list__item--active">
```

Comment:
```suggestion
          <span class="site-list__navigation site-list__navigation--active">
```

Модифицируем элемент `navigation`

И Йоксель тут находит ошибку
![image](https://github.com/htmlacademy-adaptive/1919993-cat-energy-1/assets/30808818/2c49cd91-4384-4576-b508-7aca987c6c12)